tests/conftest.py

```diff
@@ -101,3 +101,9 @@ def pointset():
     rng = default_rng(0)
     points = rng.random((10, 3))
     return pyvista.PointSet(points)
+
+
+def pytest_addoption(parser):
```

Comment:
Needed to move these up a level.  Previously these would only be set when calling `pytest tests/plotting ...` but but when ``pytest tests ...`.  I do not see a downside to allowing this more flexible method for all options.

Reply:
Agreed.